pkg/router/template/template_helper_test.go

```diff
@@ -767,3 +767,37 @@ func TestProcessEndpointsForAlias(t *testing.T) {
 		router.DeleteEndpoints(suKey)
 	}
 }
+
+func TestClipHAProxyTimeoutValue(t *testing.T) {
+	testCases := []struct {
+		value    string
+		expected string
+	}{
+		{
+			value:    "",
+			expected: "",
+		},
+		{
+			value:    "10s",
+			expected: "10s",
+		},
+		{
+			value:    "10d",
+			expected: "10d",
+		},
+		{
+			value:    "100d",
+			expected: haproxyMaxTimeout,
+		},
+		{
+			value:    "1000h",
```

Comment:
And a [test] case without any units defaults to milliseconds?

Reply:
> And a [test] case without any units defaults to milliseconds?

haproxy defaults to milliseconds: 

```
timeout client <timeout>
timeout clitimeout <timeout> (deprecated)
  Set the maximum inactivity time on the client side.
  May be used in sections :   defaults | frontend | listen | backend
                                 yes   |    yes   |   yes  |   no
  Arguments :
    <timeout> is the timeout value specified in milliseconds by default, but
              can be in any other unit if the number is suffixed by the unit,
              as explained at the top of this document.

```

Reply:
If no time unit is specific, HAProxy will assume milliseconds.

Added a test to highlight this.